app/forms/steps/outgoings/board_and_lodgings_form.rb

```diff
@@ -0,0 +1,52 @@
+module Steps
+  module Outgoings
+    class BoardAndLodgingsForm < Steps::BaseFormObject
+      attribute :board_amount, :pence
+      attribute :food_amount, :pence
+      attribute :frequency, :value_object, source: PaymentFrequencyType
+      attribute :payee_name, :string
+      attribute :payee_relationship_to_client, :string
+
+      validates :board_amount, presence: true
+      validates :food_amount, presence: true
+      validates :frequency, inclusion: { in: PaymentFrequencyType.values }
+      validates :payee_name, presence: true
+      validates :payee_relationship_to_client, presence: true
+
+      def outgoings_payment
+        @outgoings_payment ||= crime_application.outgoings_payments.find_by(
+          payment_type: OutgoingsPaymentType::BOARD_AND_LODGING.to_s
+        )
+      end
```

Comment:
Is this method definitely needed? I think it is only used by the `persist` method now?